Cleanse/ScopedBinder.swift

```diff
@@ -27,6 +27,10 @@ public struct Binder<S: Cleanse._ScopeBase> : BinderType, WrappedBinder {
     }
 
     public typealias Scope = S
+
+    public init(binder: BinderBase) {
```

Comment:
Here's the second init that is failing the test. Definitely need to delete this.